lib/avo/engine.rb

```diff
@@ -72,6 +72,14 @@ class Engine < ::Rails::Engine
       require_relative "../generators/model_generator"
     end
 
+    LOCALES_PATH = File.join(__dir__, "config", "locales")
+
+    config.i18n.load_path += if File.directory?(LOCALES_PATH)
+                               Dir[Rails.root.join("config", "locales", "*.{rb,yml}")]
+                             else
+                               Dir[Avo::Engine.root.join("lib", "generators", "avo", "templates", "locales", "*.{rb,yml}")]
+                             end
```

Comment:
There are a few things here:

1. the `LOCALES_PATH` does not check fo the parent app locales dir. so we're essentially checking the wrong path.
2. I don't know if this is a good strategy, loading one locale or another. I think we should add the to rails and say something like, "if you have it in the parent app, use that; if not, fallback to ours".

I wrapper everything up in an initializer to have it run in the right point in the boot-up flow.

```suggestion
    initializer "avo.locales" do |app|
      I18n.load_path += Dir[Avo::Engine.root.join("lib", "generators", "avo", "templates", "locales", "*.{rb,yml}")]
      I18n.load_path += Dir[Rails.root.join("config", "locales", "*.{rb,yml}")]
    end
```